src/common/responseHelpers.test.ts

```diff
@@ -0,0 +1,23 @@
+import { DataFrame, FieldType } from '@grafana/data';
+import { firstStringFieldToMetricFindValue } from './responseHelpers';
+
+describe('firstStringFieldToMetricFindValue', () => {
+  it('should find a string field value', () => {
+    const frame: DataFrame = {
```

Comment:
May be worth mocking the `DataFrame` type as I imagine this will be used elsewhere. Also helps with consistency across plugins.